src/patternfly/components/Nav/nav.scss

```diff
@@ -394,58 +217,37 @@
   }
 
   .pf-c-nav__link {
-    padding-top: var(--pf-c-nav__horizontal-list-link--PaddingTop);
-    padding-bottom: var(--pf-c-nav__horizontal-list-link--PaddingBottom);
-    font-weight: var(--pf-c-nav__horizontal-list-link--FontWeight);
+    padding-top: var(--pf-c-nav__list-link--PaddingY);
+    padding-bottom: var(--pf-c-nav__list-link--PaddingY);
+    font-weight: var(--pf-c-nav__list-link--FontWeight);
     color: var(--pf-c-nav__horizontal-list-link--Color);
     white-space: nowrap;
-    background-color: var(--pf-c-nav__horizontal-list-link--BackgroundColor);
 
     @media screen and (min-width: $pf-global--breakpoint--lg) {
-      --pf-c-nav__horizontal-list-link--PaddingBottom: var(--pf-c-nav__horizontal-list-link--lg--PaddingBottom);
+      --pf-c-nav__list-link--PaddingY: var(--pf-c-nav__horizontal-list-link--lg--PaddingBottom);
```

Comment:
can you rename `var(--pf-c-nav__horizontal-list-link--lg--PaddingBottom)` to `var(--pf-c-nav__horizontal-list-link--lg--PaddingY)`?

Reply:
after looking at it some more, looks like the original padding-top and padding-bottom were different, so this shouldn't be a shared var. And only the padding-bottom changes in this MQ.

Reply:
Attached the design which shows that top and bottom are the same, so changing the MQ to just change padding-bottom

<img width="643" alt="screen shot 2018-12-11 at 1 38 23 pm" src="https://user-images.githubusercontent.com/20118816/49822070-1534f700-fd4a-11e8-80ce-84b0b0f15394.png">

Reply:
Ah, thanks! My sketch app expired, but I just tried re-installing and that works, so I was able to pull this up, too. Doesn't look to me like the padding-bottom changes between desktop/mobile? Just measuring it  And in mobile, the underline is wider than the text?

Reply:
Yeah I didn't see the padding-bottom changing either, maybe the design has been updated, but I'll remove that media query.

Reply:
Oh yes good catch, looks like its missing a large paddingRight and PaddingLeft

Reply:
Personally I'm inclined to just leave what's already there or verify with design first that our implementation is wrong. I know some changes are made by communicating to the developer but don't make their way back into the master sketch file.

Reply:
Michael confirmed that the padding needs to be updated so going to update that.

Reply:
Looks like the desktop version isn't inline with the sketch file.
Here's what you have:
<img width="181" alt="screen shot 2019-01-03 at 1 58 40 pm" src="https://user-images.githubusercontent.com/287880/50653409-cf8dd780-0f5f-11e9-89b2-cd22f6e2d320.png">
Here's what in the Sketch file:
<img width="335" alt="screen shot 2019-01-03 at 2 00 18 pm" src="https://user-images.githubusercontent.com/287880/50653455-ef250000-0f5f-11e9-936c-42b1badad7a3.png">